projects/api/test/product/product.controller.spec.ts

```diff
@@ -106,14 +116,24 @@ describe('ProductController', () => {
     describe('findAllByName2', () => {
         it('should return an array of products by name', async () => {
             const products: Product[] = [{
```

Comment:
I saw this code pattern repeatedly, so Is it not better to have a constant file to add all the constant variables? or maybe a utility is better?